CMakeLists.txt

```diff
@@ -117,6 +117,24 @@ set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 set(EXECUTABLE_OUTPUT_PATH ${CMAKE_CURRENT_BINARY_DIR})
 set(LIBRARY_OUTPUT_PATH ${CMAKE_CURRENT_BINARY_DIR})
 
+if (UNIX)
+  find_program(CCACHE_FOUND ccache)
+  if(CCACHE_FOUND)
```

Comment:
space after if

Reply:
fixed. Should I resolve discussion myself, or is it responsibility of the reviewer?